lib/ansible/plugins/doc_fragments/action_common_attributes.py

```diff
@@ -41,20 +49,38 @@ class ModuleDocFragment(object):
     facts:
       description: Action returns an ``ansible_facts`` dictionary that will update existing host facts
       support: none
+    forced_local:
+      description: The connection itself is passed to the action while the code is still executed on the controller
+      support: none
+    forced_action_plugin:
+      description: This action uses a specific action plugin instead of 'normal' or matching by name
+      support: none
+      action_plugin: none
```

Comment:
Should `ansible-doc` disallow extra arbitrary fields besides these (`description`, `support`, `membership`, `action_plugin`)? It might help keep the documentation standardized.

Reply:
eventually, right now this is an 'evolving' structure and I'm still discovering 'the needs'